org.gaphor.Gaphor.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
It should be put into upstream repo so other distributions can benefit too.

Reply:
Ok, removed, thanks :+1: